package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "build:docs": "cd docs && yarn gatsby build --prefix-paths",
     "build:docs:preview": "cd docs && yarn gatsby build",
-    "prepare": "tsc && rollup -c",
+    "prepare": "rm -rf docs/static && mkdir docs/static && tsc && rollup -c && cp app/assets/javascripts/primer_view_components.js docs/static/primer_view_components.js",
```

Comment:
Is there anything else in docs/static that this command is blowing away?

Reply:
Nope, this is a new directory!